### PR TITLE
[ticket/14345] Check if description exists in message ary

### PR DESCRIPTION
**Pull request by marc1706**
Ticket: https://tracker.phpbb.com/browse/PHPBB3-14345

### CODE REVIEW

File: phpBB/phpbb/install/helper/iohandler/cli_iohandler.php [hunk 1]
```diff
@@ -126,7 +126,8 @@ public function add_error_message($error_title, $error_description = false)
 		$this->io->newLine();
 
 		$message = $this->translate_message($error_title, $error_description);
-		$this->io->error($message['title'] . "\n" . $message['description']);
+		$message_string = $message['title'] . "\n" . (!empty($message['description']) ? $message['description'] : '');
```
Comment (Nicofuma) [inline phpBB/phpbb/install/helper/iohandler/cli_iohandler.php]: the EOL char is needed only when there is a description IIRC. You should move it to the ternary

Comment (marc1706) [inline phpBB/phpbb/install/helper/iohandler/cli_iohandler.php]: Ok, will do.

**Comment by marc1706** [comment]
@Nicofuma updated

**Comment by Nicofuma** [comment]
:+1:

**Pull request merged**